src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -461,6 +461,9 @@ def getClusterAd(self):
 
         ad['JobLeaseDuration'] = classad.ExprTree('isUndefined(MachineAttrMaxHibernateTime0) ? 1200 : MachineAttrMaxHibernateTime0')
 
+        ad['PeriodicRemove'] = classad.ExprTree('( JobStatus =?= 5 ) && ( time() - EnteredCurrentStatus > 10 * 60 )')
+        ad['PeriodicRemoveReason'] = "Job automatically removed for being in Held status"
```

Comment:
maybe only set the PeriodicRemoveReason when the job is actually removed ? Avoid confusing someone who looks at job ads one day and finds this reason set. The main reason to set a PeriodicRemoveReason is in case the PeriodicRemove is an OR or various conditions and you want to record which one triggered it, e.g. in CRAB we want to know if job was removed because it was using too much time, or too much memory or...